examples/python_side_metrics.py

```diff
@@ -20,6 +21,14 @@
 from typing import List
 from wsgiref.simple_server import make_server
 
+from prometheus_client import (  # type: ignore
+    CONTENT_TYPE_LATEST,
+    CollectorRegistry,
+    Counter,
+    generate_latest,
+    multiprocess,
+)
+
```

Comment:
This part doesn't need to follow the PEP standard. Because the demo will show how to set the `PROMETHEUS_MULTIPROC_DIR` environment variables. Can ignore the `wrong-import-position` for this case.